src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -0,0 +1,345 @@
+from copy import deepcopy
+from typing import Literal, Optional
+
+import numpy as np
+from pydantic import FilePath, validate_call
+from pynwb import NWBFile
+from pynwb.image import ImageSeries
+
+from .video_utils import VideoCaptureContext
+from ....basedatainterface import BaseDataInterface
+from ....tools import get_package
+from ....tools.nwb_helpers import get_module
+from ....utils import dict_deep_update, get_base_schema, get_schema_from_hdmf_class
+
+
+class ExternalVideoInterface(BaseDataInterface):
+    """Data interface for writing videos as external_file ImageSeries."""
+
+    display_name = "Video"
+    keywords = ("video", "behavior")
+    associated_suffixes = (".mp4", ".avi", ".wmv", ".mov", ".flx", ".mkv")
+    # Other suffixes, while they can be opened by OpenCV, are not supported by DANDI so should probably not list here
+    info = "Interface for handling standard video file formats."
```

Comment:
```suggestion
    info = "Interface for handling standard video file formats and write them as ImagingSeries with external_files."
```
or something like that.

I think this is displayed on the guide

Reply:
Done!